src/main/java/com/synopsys/integration/alert/web/config/JobConfigController.java

```diff
@@ -60,49 +64,49 @@
 @RequestMapping(JobConfigController.JOB_CONFIGURATION_PATH)
 public class JobConfigController extends BaseController {
     public static final String JOB_CONFIGURATION_PATH = ConfigController.CONFIGURATION_PATH + "/job";
-    private static final String[] REQUIRED_PERMISSION_KEYS = { PermissionKeys.CONFIG_DISTRIBUTION_CHANNEL_EMAIL.getDatabaseKey(),
-        PermissionKeys.CONFIG_DISTRIBUTION_CHANNEL_SLACK.getDatabaseKey(),
-        PermissionKeys.CONFIG_DISTRIBUTION_PROVIDER_BLACKDUCK.getDatabaseKey(),
-        PermissionKeys.CONFIG_DISTRIBUTION_PROVIDER_POLARIS.getDatabaseKey()
-    };
     private final Logger logger = LoggerFactory.getLogger(JobConfigController.class);
 
     private final JobConfigActions jobConfigActions;
     private final ResponseFactory responseFactory;
     private final ContentConverter contentConverter;
     private final AuthorizationManager authorizationManager;
+    private final DescriptorAccessor descriptorAccessor;
 
     @Autowired
-    public JobConfigController(final JobConfigActions jobConfigActions, final ResponseFactory responseFactory, final ContentConverter contentConverter, final AuthorizationManager authorizationManager) {
+    public JobConfigController(final JobConfigActions jobConfigActions, final ResponseFactory responseFactory, final ContentConverter contentConverter, final AuthorizationManager authorizationManager,
+        DescriptorAccessor descriptorAccessor) {
         this.jobConfigActions = jobConfigActions;
         this.responseFactory = responseFactory;
         this.contentConverter = contentConverter;
         this.authorizationManager = authorizationManager;
+        this.descriptorAccessor = descriptorAccessor;
     }
 
     @GetMapping
     public ResponseEntity<String> getJobs() {
-        if (!authorizationManager.anyReadPermission(REQUIRED_PERMISSION_KEYS)) {
-            return responseFactory.createForbiddenResponse();
-        }
-        final List<JobFieldModel> models = new LinkedList<>();
         try {
+            final Set<String> descriptorNames = descriptorAccessor.getRegisteredDescriptors()
+                                                    .stream()
+                                                    .map(RegisteredDescriptorModel::getName)
+                                                    .filter(name -> name.startsWith("channel") || name.startsWith("provider"))
```

Comment:
This is a very risky check. Going forward, we probably want to remove these sort of checks wherever we see them and use some other method.

Reply:
To add to this, Eric right now is talking to Paulo about calling his msteams channel something that doesn't much the assumed state above. If we have new people adding content, we'll want to find a better solution to this (Like checking the actual descriptor type).

Reply:
there is the DescriptorType returned from the getType().  We should filter on that rather than name.